db/schema.rb

```diff
@@ -0,0 +1,27 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2021_07_07_155814) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "users", force: :cascade do |t|
```

Comment:
Don't you think it's better to use UUID as the primary key?

Reply:
@EgorBarakhov  Why?

Reply:
> - UUID values are unique across tables, databases, and even servers that allow you to merge rows from different databases or distribute databases across servers.
- UUID values do not expose the information about your data so they are safer to use in a URL. For example, if a customer with id 10 accesses his account via http://www.example.com/customers/10/ URL, it is easy to guess that there is a customer 11, 12, etc., and this could be a target for an attack.
- UUID values can be generated anywhere that avoid a round trip to the database server. It also simplifies logic in the application. For example, to insert data into a parent table and child tables, you have to insert into the parent table first, get generated id and then insert data into the child tables. By using UUID, you can generate the primary key value of the parent table up front and insert rows into both parent and child tables at the same time within a transaction.

source: https://www.mysqltutorial.org/mysql-uuid/

Reply:
Looks good, but I'm not going to use user IDs in URLs and going to build a monolith app.
@EgorBarakhov Do you know if UUID is as easy to use as simple integer IDs in Rails?
Do I need some extra actions to maintain UUIDs?

Reply:
I'll try to follow this article https://pawelurbanek.com/uuid-order-rails